src/components/Map/LegendControls.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div
     class="resizable draggable-container"
+    :class="getHidden.title && getHidden.topMenus ? 'hidden-top' : ''"
```

Comment:
Let's use object-syntax for this class binding? https://v2.vuejs.org/v2/guide/class-and-style#Object-Syntax
`{ 'hidden-top': getHidden.title && getHidden.topMenus }`